# To compile the mini-installer, a 7z archive of the browser has to be made.
# Google's 7z executable gets purged, so we have to use the system's 7z executable.

--- a/chrome/installer/mini_installer/BUILD.gn
+++ b/chrome/installer/mini_installer/BUILD.gn
@@ -191,7 +191,6 @@ action("mini_installer_archive") {
     "//chrome:chrome_dll",
     "//chrome/browser/extensions/default_extensions",
     "//chrome/chrome_elf",
-    "//chrome/common/win:eventlog_provider",
     "//chrome/installer/setup",
     "//third_party/icu:icudata",
   ]